### PR TITLE
More robust CUISequenceSimpleItem loading to prevent crashes when loading game tutorials 

### PR DESCRIPTION
**Pull request by sobkas**
Added NULL-checks, skipping not working subitems, increased verbosity of messages
Starting New Game Clear Sky with this change works
Need to also test COP

### CODE REVIEW

File: src/xrGame/ui/UIGameTutorial.cpp
Comment (Xottab-DUTY) [inline src/xrGame/ui/UIGameTutorial.cpp:1]: Changes to this file totally not needed.
If there's no `game_tutorials.xml`, an assert will be triggered in `CUIXml::Load`. Other changes that add additional logging actually pollute the log.

File: src/xrGame/ui/UIGameTutorialSimpleItem.cpp [hunk 1]
```diff
@@ -42,8 +42,12 @@ CUIWindow* find_child_window(CUIWindow* parent, const shared_str& _name)
     auto _I = wl.begin();
     auto _E = wl.end();
     for (; _I != _E; ++_I)
-        if ((*_I)->WindowName() == _name)
+    {
+        if ((*_I)->WindowName().equal(_name))
+        {
             return (*_I);
+        }
+    }
```
Comment (Xottab-DUTY) [inline src/xrGame/ui/UIGameTutorialSimpleItem.cpp]: This is not needed too

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]

File: src/xrGame/ui/UIGameTutorialSimpleItem.cpp (hunk 1 above)
Comment (sobkas) [reply src/xrGame/ui/UIGameTutorialSimpleItem.cpp]: Fixed

**Comment by sobkas** [comment]
Fixed coding style problems

File: src/xrGame/ui/UIGameTutorial.cpp [hunk 2]
```diff
@@ -114,6 +114,10 @@ bool CUISequencer::Start(LPCSTR tutor_name)
         Msg("! can't find tutorial [%s]", tutor_name);
         return false;
     }
+    else
+    {
+        Msg("found tutorial [%s]", tutor_name);
+    }
```
Comment (Xottab-DUTY) [inline src/xrGame/ui/UIGameTutorial.cpp]: This is not needed too.
```suggestion
```

Comment (sobkas) [reply src/xrGame/ui/UIGameTutorial.cpp]: Fixed

**Comment by Xottab-DUTY** [comment]
So, I've found the real culprit of this crash. But we still need some changes to `CUISequenceSimpleItem` to make it more robust!
Let me do them in this PR 🙂

**Comment by Xottab-DUTY** [comment]
Thank you!

**Pull request merged**